EX05/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const fs = require('fs')
+const path = './tasks.json';
+const bodyParser = require('body-parser');
+const PORT = 8080;
+let app = express();
+app.use(bodyParser.json());
+
+
+app.get('/tasks', (req, res) => {
+    console.log("Got 'Get' request");
+	let showTasks;
+	fs.readFile(path, (err, content)=> {
+		if (err){
+			     throw err;				
```

Comment:
It is better to send a clear error to the user so he will know there was a problem in the server.